contracts/mixnet/src/contract.rs

```diff
@@ -29,11 +31,19 @@ pub const INITIAL_GATEWAY_DELEGATION_REWARD_RATE: u64 = 110;
 pub const INITIAL_MIXNODE_ACTIVE_SET_SIZE: u32 = 100;
 pub const INITIAL_GATEWAY_ACTIVE_SET_SIZE: u32 = 20;
 
+pub const INITIAL_REWARD_POOL: u128 = 250_000_000_000_000;
```

Comment:
This comment is not really about the code, but something to remember during deployment (so that one more person is aware) - we need to actually put those tokens into the contract on initialisation